variables.tf

```
@@ -118,3 +118,9 @@ variable "transition_to_ia" {
   description = "Indicates how long it takes to transition files to the IA storage class. Valid values: AFTER_7_DAYS, AFTER_14_DAYS, AFTER_30_DAYS, AFTER_60_DAYS and AFTER_90_DAYS"
   default     = ""
 }
+
+variable "allowed_cidr_blocks" {
+  type        = list(string)
+  default     = []
+  description = "The whitelisted CIDRs which to allow `ingress` traffic to the DB instance"
```

One thing that the industry is trying to do is remove or replace terms like `whitelisted` and `blacklisted`. 

```suggestion
  description = "The CIDRs which to allow `ingress` traffic to the DB instance"
```

In this case, we can omit the word and maintain the same clarity.